lib/scripts/users_list.rb

```diff
@@ -0,0 +1,32 @@
+require 'csv'
+
+class UsersList
+  attr_accessor :csv_file
+
+  def initialize
+    @csv_file = CSV.open(File.join(Rails.root, "/lib/scripts/results/users_list.csv"), "w") do |csv|
+      csv << ['Formal Name', 'netID', 'email', 'Current sign in', 'Last sign in']
```

Comment:
I'm not sure if the case for each header is intentional, but if we can I think we should capitalize each header.

Reply:
Sure, good idea.